select_assignment2_flicks/Model/Movie.swift

```diff
@@ -8,17 +8,27 @@
 
 import UIKit
 import SwiftyJSON
+import RealmSwift
+
+class MovieRealm : Object {
+    var id: String = ""
```

Comment:
@chug2k: I cannot catch your idea, realm doesn't support struct so it must be class right? Or do you mean using dynamic keyword?